juspay/hyperswitch
# fix(events): fix event generation for paymentmethods list

## Type of Change



## Description

- Add payment_id field in payment methods list

### Additional Changes





## Motivation and Context



## How did you test it?

1. create a payment with confirm false
2. call `List payment methods for a Merchant` API via postman

## Checklist
